package.json

```diff
@@ -56,7 +56,7 @@
     "typescript": "^4.4.4"
   },
   "peerDependencies": {
-    "react": "16"
+    "react": "^16.8.0 || ^17.0.2"
```

Comment:
If we happen to have any other dependency that have react in 17.0.1 or 17.0.0 as peer dependency. Won't this cause trouble when trying to install it? Wouldn't it be best to use 17.0.x?

Reply:
yes, it may cause a problem for those who want to install react, but there is no reason to not install react 17.0.2 instead of 17.0.0, since the difference between those two versions are bugfix and we don't want to install broken versions.

ref: [react 17.0.2 release](https://github.com/facebook/react/blob/main/CHANGELOG.md#1702-march-22-2021)